content/ja/docs/concepts/services-networking/service-traffic-policy.md

```diff
@@ -0,0 +1,60 @@
+---
+reviewers:
+- maplain
+title: サービス内部トラフィックポリシー
+content_type: concept
+weight: 45
+---
+
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.21" state="alpha" >}}
```

Comment:
https://github.com/kubernetes/kubernetes/pull/103462
I think ServiceInternalTrafficPolicy field to Beta from v1.22

Reply:
I see. However, the original webpage has not changed yet(https://kubernetes.io/docs/concepts/services-networking/service-traffic-policy/). I think that alpha is correct.

Reply:
@KobayashiD27 Could you create an issue to this repository that says this should be changed in the upstream?